### DISCUSSION TITLE
Speaker group object option: Volume sliders

### DISCUSSION THREAD

**Discussion by jansenfd** [kalkih/mini-media-player#684] [Ideas]
I would like to opt an idea.
When grouping speakers using the "speaker group object" is it possible to add individual volume sliders in the same list that pops up?

**Comment by ZNicoZ** [comment]
There is already a PR from two years ago, doesnt seem to be easy.
For sonos, I´d like to have the same volume management as in the sonos app:

Group-Volume: changes volume of all speakers in group but not syncing them  (the relative difference in volume between speakers should stay)
Speaker 1 Volume: only changes volume of speaker 1
Speaker 2 Volume: only changes volume of speaker 2

**Comment by Dankeyking** [comment]
Is there now a possible way to set the volume of all speakers in the media player? if I use the mute button it only mutes the master and not any other device also with the loudness. Or am I missing something, i'm new to this
